### PR TITLE
Add support for full path in the identifier part

### LINKED ISSUE #32: Add support for full path in the identifier part
The spec for web-did has been updated to include an optional path to the did document.
The requirements for this are described in the discussion on w3c-ccg:
https://github.com/w3c-ccg/did-method-web/pull/5

Fixing this may affect or even close #17

### PR DESCRIPTION
**Pull request by ajunge**
Resolves #32 #33

### CODE REVIEW

**Comment by ramxis** [comment]
hi can we merge this pull request ? We need to support user dids (i.e did:web:user..) for our project.

File: src/__tests__/uport.test.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ describe('web did resolver', () => {
     didResolver = new Resolver(webDidResolver)
   })
 
-  it('resolves document', () => {
+  it.skip('resolves document', () => {
```
Comment (mirceanis) [inline src/__tests__/uport.test.ts:15]: this test should not be skipped.
```suggestion
  it('resolves document', () => {
```

**Review by mirceanis — CHANGES_REQUESTED** [review]
Thank you for the contrib.
There's a test that seems to be skipped which needs to be fixed before being able to merge this.

**Comment by mirceanis** [comment]
@ramxis not yet.

There was a skipped test which seems to be failing. We have to fix that before being able to merge.

**Comment by mirceanis** [comment]
Update: It looks like it's not a problem with the library but the endpoint it was hitting for that test. I'll revert it and trigger a new release.

**Review by mirceanis — APPROVED** [review]

**Pull request merged**